internal/dataplane/kongstate/route.go

```diff
@@ -197,15 +197,15 @@ func (r *Route) overrideMethods(log logrus.FieldLogger, anns map[string]string)
 		} else {
 			// if any method is invalid (not an uppercase alpha string),
 			// discard everything
-			log.WithField("kongroute", r.Name).Errorf("invalid method: %v", method)
+			logger.V(util.ErrorLevel).Error(nil, "invalid method", "route_name", r.Name, "method", method)
```

Comment:
Maybe. It's ignored but the value is still available. Whether that value is printed is implementation-dependent. zapr at least [only includes it for Info](https://github.com/go-logr/zapr/blob/fe8a3ccb2b17db0a1602e5a39b6221b40a7ed4fb/zapr.go#L274-L291).

I originally included it somewhat arbitrarily to make log calls consistent, and because there used to be some `V(util.DebugLevel).Error()` calls from `logrus.Errorf("something %w", err)` calls at level debug. The latter I've since converted to `Info()` as I think we cared more about the threshold and happened to have an error from upstream that we could include.